apps/docs/docs/api/browser-wallets.mdx

```diff
@@ -120,6 +134,7 @@ Gets a list of addresses for the browser wallet if authorized.
 
 **Throws**:
 
+- `BrowserWalletNetworkMismatchError` if Magic Eden Wallet is not used on `mainnet`
```

Comment:
generalise this error description, but indicate that it will appear for Magic Eden Wallet

also remove `if` for this or add `if` to the rest, let's pick one format

Reply:
Dropped the `if`.